### ISSUE TITLE
HTML output file - build throws exception

### ISSUE THREAD

**Issue by TakhirMamirov** [rfennell/vNextBuild#246]
I've tried the latest version as of Jan 18, 2018 and it worked perfectly for .md templates. Is that possible to generate the output in HTML format as well as define the template as styled HTML, not md? If I simply try to use a sample HTML from the Wiki:
```
<h1>Release notes for build $defname</h1>  
<b>Build Number</b>  : $($build.buildnumber)    <br>  
<b>Build started</b> : $("{0:dd/MM/yy HH:mm:ss}" -f [datetime]$build.startTime)     <br>  
<b>Source Branch</b> : $($build.sourceBranch)  <br>  
<h3>Associated work items</h3>  
@@WILOOP@@  
<li> <b>$($widetail.fields.'System.WorkItemType') $($widetail.id)</b> [Assigned by: $($widetail.fields.'System.AssignedTo')]     $($widetail.fields.'System.Title') 
@@WILOOP@@  
<h3>Associated change sets/commits</h3>  
@@CSLOOP@@  
<li> <b>ID $($csdetail.changesetid)$($csdetail.commitid)</b> $($csdetail.comment)    
<hr>  
@@CSLOOP@@
```
then I get the following exception:

`##[error]System.Management.Automation.ParameterBindingValidationException: Cannot bind argument to parameter 'Message' because it is null.
`

**Comment by rfennell** [comment]
I have just tried you template in my validation build (a build not a release) against a Git project.

It worked without error and I got a valid HTML file. Does my test match your usage

The usual reason for this error is a incorrect parameter called on an object or a missing object. I did add some extra logging in the 2.13 release, so if you return the build with the system.debug variable set to true should show you, on the line before the error, what it failed to parse.
